README.md

```diff
@@ -173,15 +173,15 @@ import { Pinecone } from '@pinecone-database/pinecone';
 const pc = new Pinecone();
 
 await pc.createIndex({
-  name: 'serverless-index',
-  dimension: 1536,
-  spec: {
-    serverless: {
-      cloud: 'aws',
-      region: 'us-west-2',
+    name: 'serverless-index',
+    dimension: 1536,
+    spec: {
+        serverless: {
+            cloud: 'aws',
+            region: 'us-west-2',
+        }
```

Comment:
These types of changes come automatically with `npm run format`

Reply:
This is a case where I'd suggest holding off on the formatting. It can seem innocuous, but given that these changes will live on the release branch for some period of time and then later merge to main, the formatting updates can make the merge to main difficult if any of the underlying code changes on main between now and then. 

Solvable with merge conflict resolution, but still more trouble than its worth IMO.

We can always do a dedicated formatting PR off of the main branch that just runs the formatter and merge that to main.

Reply:
I'm also fine leaving the formatting changes in place with this PR -- your call

Reply:
I'm pretty sure CI fails w/o these changes (I just run the CI commands themselves locally before pushing, but lemme remove these and see what happens)

Reply:
Yeah, it fails, I have to include these lints: https://github.com/pinecone-io/pinecone-ts-client/actions/runs/9490755109/job/26154849071?pr=230. Reverting.

Reply:
sounds good